### ISSUE TITLE
Issue: Cannot long press to rearrange

### ISSUE THREAD

**Issue by CorvusUrro** [PasteBar/PasteBarApp#170]
In the "Saved Clip Item" feature highlight it says "Long-press to rearrange items" however I am unable to get this feature to work and have to be in organise mode to rearrange items.

**Comment by kurdin** [comment]
@CorvusUrro Yeah, I think I removed that feature in the latest version because it did not work well with regular click functionality and I had multiple false long press action when using regular click. You can always use context menu on a clip and select  `Organize` item to enable rearrange items functionality. I don't think that Long-press is needed.
